scripts/docgen/snippets/extractSnippets.ts

```diff
@@ -147,19 +103,16 @@ const extractSnippetRangesFromFile = (sourceFile: SourceFile): SnipRange[] => {
                     matchingSnipStartIndex,
                     1
                 )[0]
-                snipRanges.push({
-                    id: parsedSnip.id,
-                    range: {
-                        start: matchingSnipStart.node,
-                        end: parsedSnip.node
-                    }
-                })
-            } else {
-                snipRanges.push({
-                    id: parsedSnip.id,
-                    range: { node: parsedSnip.node }
-                })
+                text = sourceFileText.slice(matchingSnipStart.lineNumber, i + 1)
+                text = extractTextBetweenLines(
+                    textSplitByNewLine,
+                    matchingSnipStart.lineNumber,
+                    i
```

Comment:
It's confusing to have `i` and `i + 1` in so many places. Maybe create a variable called `lineNumber` for each iteration of the loop equal to `i + 1` so it's obvious where you're using it?